### DISCUSSION TITLE
Multiple file upload and download in SFTP

### DISCUSSION THREAD

**Discussion by betacat2048** [isontheline/pro.webssh.net#1036] [Ideas]
When synchronizing cloud servers, I often need to upload or download multiple files.

Therefore, it is hoped that additional batch download options can be added to download multiple files at the same time.

Although currently WebSSH can select multiple files at the same time when uploading, its success seems to be highly dependent on a low-latency network or a direct SSH connection? In my use, once port forwarding is enabled and there is a certain network delay, there is a high probability that the upload will be interrupted after the first file transfer is completed. Therefore, I hope to add an automatic breakpoint resume function.

**Comment by isontheline** [comment]
You're right I need to improve batch uploads and downloads : https://github.com/isontheline/pro.webssh.net/issues/205

About your high-latency trouble : any error thrown? or just hang?
